test/e2e/e2e_test.go

```diff
@@ -392,7 +436,8 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 			}()
 
 			By("Waiting for all pods to complete successfully")
-			err := wait.PollImmediate(time.Second, 5*time.Minute, func() (done bool, err error) {
+			timeout := 10 * time.Minute
```

Comment:
timeout is changed due to windows test?
we could make it different for windows and linux case.

Reply:
sure, I changed it to be 10m for windows only